### PR TITLE
fix(graph): de-duplicate dorks and vulnerabilities

### LINKED ISSUE #124: feat(vizualization): display only one occurrence of a vulnerability for a target
### Expected feature

In the visualization graph, and the PDF report (Full and vulnerabilities) the same vulnerabilities, duplicated because they are found on multiple endpoints, are displayed multiple times.

##### Zoom in
image

##### Zoom out
image

This make the visualization graph unreadable.

#### Report

image

So the best way to make it more readable is to display the same vuln only once.
For the visualization, no need to display the endpoints, on the report endpoints could be listed inside the vuln section

### Alternative solutions

_No response_

### Anything else?

_No response_

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

### PR DESCRIPTION
**Pull request by psyray**
Fixes #124 
With this PR dorks and vulnerabilites are deduplicated to improve graph display on a lot of targets

Here with vulnweb.com targets
vulnweb comvulnweb com_visualisation

### CODE REVIEW

File: web/static/custom/mitch.js
Comment (AnonymousWP) [inline web/static/custom/mitch.js:1]: It has French comments.

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
I'll assign the developer team to this PR.

File: web/static/custom/mitch.js
Comment (psyray) [reply web/static/custom/mitch.js:1]: Fixed

File: web/api/serializers.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
+from collections import defaultdict
 from dashboard.models import *
 from django.contrib.humanize.templatetags.humanize import (naturalday,
-														   naturaltime)
+														naturaltime)
```
Comment (AnonymousWP) [inline web/api/serializers.py:4]: Something is going wrong here, see https://github.com/Security-Tools-Alliance/rengine-ng/blob/group-vulns-on-visualization/web/api/serializers.py.

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Review by AnonymousWP — APPROVED** [review]

**Pull request merged**